pintc/src/error/handler.rs

```diff
@@ -12,10 +12,10 @@ pub struct Handler {
 /// Contains the actual data for `Handler`.
 /// Modelled this way to afford an API using interior mutability.
 #[derive(Default, Debug)]
-struct HandlerInner {
-    /// The sink through which errors will be emitted.
-    errors: Vec<Error>,
-    // TODO: add warnings here
+pub struct HandlerInner {
```

Comment:
So this used to be private on purpose. I don't think we should make it public. Is this necessary?